app/src/main/java/gov/hhs/cdc/trustedintermediary/external/javalin/App.java

```diff
@@ -50,6 +51,7 @@ public static void main(String[] args) {
 
         try {
             app.get(HEALTH_API_ENDPOINT, ctx -> ctx.result("Operational"));
+            app.get(ROOT_API_ENDPOINT, ctx -> ctx.result("Operational"));
```

Comment:
Maybe as a future improvement, this  can serve as the landing endpoint for documentation, or provide other information like API version or build info. I know we already have the swagger endpoint, but it can be a possibility.